app/views/layouts/application.html.erb

```diff
@@ -39,6 +39,7 @@
   <body>
     <%= content_for?(:body) ? yield(:body) : yield %>
     <%= render 'shared/ajax_modal' %>
+    <script src="//cdn.jsdelivr.net/npm/clipboard@2.0.6/dist/clipboard.js"></script>
```

Comment:
We weren't able to package this in?

Reply:
Add a comment: There were bugs when we tried to include this webpack. We can add a link to the comment chain in this PR.